tests/livesim_tests/livesim_topology.yaml

```diff
@@ -0,0 +1,261 @@
+#
+# prepare topology
+#
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: default
+  labels:
+    ns: def
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ns1
+  labels:
+    ns: ns1
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deployment-A
+  labels:
+    dep: A
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      dep: A
+  template:
+    metadata:
+      labels:
+        dep: A
+    spec:
+      containers:
+        - name: my-test-app
+          image: nginx
+          ports:
+            - containerPort: 80
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deployment-B
+  labels:
+    dep: B
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      dep: B
+  template:
+    metadata:
+      labels:
+        dep: B
+        app: appA
+        release: v22b
+    spec:
+      containers:
+        - name: my-test-nginx
+          image: nginx
+          ports:
+            - containerPort: 80
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-dep-a
+spec:
+  podSelector:
+    matchLabels:
+      dep: A
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+        - podSelector:
+            matchLabels:
+              dep: B
+  egress:
+    - to:
+        - podSelector:
+            matchLabels:
+              dep: B
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-dep-b
+spec:
+  podSelector:
+    matchLabels:
+      dep: B
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+        - podSelector:
+            matchLabels:
+              dep: A
+  egress:
+    - to:
+        - podSelector:
+            matchLabels:
+              dep: A
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default-deny-egress
+  namespace: default
+spec:
+  podSelector:
+    matchLabels: {}
+  policyTypes:
+  - Egress
+---
+#
+# Kube DNS case
+#
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-dns-access
+spec:
+  podSelector:
+    matchLabels: {}
+  policyTypes:
+  - Egress
+  egress:
+  - to:
+    - namespaceSelector:
+        matchLabels:
+          name: kube-system
```

Comment:
I don't think we can infer just from a namespace `kube-system` that the missing pod is the `coredns` pod. I think we can only infer based on labelselector matching this pod (`k8s-app: kube-dns`).
`kube-system`  may have other pods being referenced in policies.

Reply:
users sometimes only refer to the namespace to allow connectivity to kube-dns without referring to the pod itself.